include/alpaka/core/Common.hpp

```diff
@@ -55,9 +55,9 @@
 //-----------------------------------------------------------------------------
 // CUDA device architecture detection
 #if defined(__CUDA_ARCH__)
```

Comment:
@BenjaminW3 shouldn't we add a BOOST guard here? e.g. < 1.58

Reply:
Yes, but I would like to keep those changes separate. We should do this when testing the 1.58 RC.

Reply:
but if we add the guard already alpaka will not break with boost > 1.57, no?

Reply:
you could also just a `#ifndef BOOST_ARCH_PTX` which is probably the best solution.

Reply:
We can not be sure that it will really be part of 1.58 or that it works the way we intended to. Therefore, the check could also make it break as well.

Reply:
The ifndef is the solution I had in mind when doing this renaming.

Reply:
@BenjaminW3 you pushed the PR again but the change with the `ifndef` is not in yet. Did you for get to `git add` the changed file first?

Reply:
Oh, I did not want to do the ifndef thing within this PR. I just had to fix a merge conflict with the merged AtomicTest fixes.